test/integration_test.js

```diff
@@ -22,7 +22,11 @@ describe("Integration tests", () => {
   var sandbox, server, tasks;
   const MAX_ATTEMPTS = 50;
 
-  function startServer(env={}) {
+  function startServer(env) {
```

Comment:
Just realized that `Object.assign({}, undefined)` doesn't raise any error — which is a little weird but, oh well. So scratch that comment we're *safe* here.